### DISCUSSION TITLE
Stations overview on public page

### DISCUSSION THREAD

**Discussion by vladosam** [AzuraCast/AzuraCast#6943] [General]
Hi, is there option to show all configured stations on one public page like stations overview on dashboard?

**Comment by Moonbase59** [comment]
No. Resellers and station ops that run testing stations would probably not want that everyone can have a look. But you can easily do such a thing using the API, on your own station web page:

A little example:

![Winterzeit ist Radiozeit | Nite Radio – Mozilla Firefox_003](https://github.com/AzuraCast/AzuraCast/assets/3706922/4c1d99e1-e93f-464c-af3e-b2388f450861)
